### ISSUE TITLE
Non-existing console method used in printer

### ISSUE THREAD

**Issue by filmaj** [architect/hydrate#19]
```➜ arc hydrate
         app ⌁ arctonic
      region ⌁ us-west-2
     profile ⌁ default
     version ⌁ Architect CLI 6.0.20
         cwd ⌁ /Users/maj/src/arctonic

⚬ Hydrating dependencies: src/http/get-api-notes [npm ci]
/Users/maj/src/cli/node_modules/@architect/hydrate/src/_printer.js:28
    if (!quiet) console.err(err)
                        ^

TypeError: console.err is not a function
    at Object.done (/Users/maj/src/cli/node_modules/@architect/hydrate/src/_printer.js:28:25)```

**Comment by ryanblock** [comment]
Woops! I'm on it.

**Comment by ryanblock** [comment]
Patched in `hydrate 1.2.3`
